daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -1109,11 +1101,11 @@ private[lf] object Speedy {
 
   private[speedy] sealed abstract class Control
   object Control {
+    final case object WeAreUnset extends Control
     final case class Expression(e: SExpr) extends Control
     final case class Value(v: SValue) extends Control
-    final case class WeAreUnset() extends Control
-    final case class WeAreComplete() extends Control
-    final case class WeAreHungry(res: SResult) extends Control
```

Comment:
RIP
`Control.{WeAreComplete,WeAreHungry}`
2022-08-04 - 2022-08-05

Reply:
This is very funny! :smile: